doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,321 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
+
+<a name="9.5.1.2"></a>
+#### 9.5.1.2 Expectations and Assumptions of Field Trials
+
+CNTT expects to exit the trial with either validation of RI-1 and RC-1 or a set of actions to close gaps. Through the community continuous improvement processes, a badging process that is mutually beneficial to operators and suppliers will be defined/refined. CNTT assumes that performance testing is not included in the field trial.
+
+<a name="9.5.1.3"></a>
+#### 9.5.1.3 Pre-trials activities
+
+Prior to the beginning of field trials, the CNTT community will define an operational plan, secure resource, and provide all designated contact information required to support the trials participants.  As the results of the trails may produce data and information that could be considered sensitive by participants, CNTT will establish standard data set requirements and secure collection methods to ensure participant privacy protection.
+
+<a name="9.5.1.4"></a>
+#### 9.5.1.4 Expectations of Trials Participants
+
+Trials participants will be expected to commit to establishing an RA-1 compliant NFVi through means that are most appropriate for the participant. 
+
+First, the participant will secure appropriate environment space from pre-existing space, newly built space or securing LaaS.  The environment can exist in private or community hardware infrastructure.
+
+Second, the participant will build/setup/configure the environment space via their preferred method. This can include the use of a cookbook, automated install, and/or build from RA-1/RI-1 requirements and specifications. CNTT RI-1 Chapter 3 (https://github.com/cntt-n/CNTT/blob/master/doc/ref_impl/cntt-ri/chapters/chapter03.md ) documentation provides the matching RI-1 requirements for the build.
+
+<a name="9.5.1.5"></a>
+#### 9.5.1.5 Expectation 2: Execute the RC-1 Test suite
+
+CNTT will provide the participants with the community RC-1 Test suite, found at this Link: https://github.com/cntt-n/CNTT/blob/master/doc/ref_impl/cntt-ri/chapters/chapter04.md. The participants will execute test cases per instructions and record the quantitative results.
+
+Test case suite should be executed successfully at least 3 times. Three times represents the recommended number of test suite runs to eliminate false positives in results.  A triage process will be used to determine and manage root cause analysis of failures. If the failures are determined to be issues with the participant’s RI, CNTT will convey the issues to the RI and make available SMEs to assist the participant in resolving the issues. When failures are deemed to be caused by an issue or gap in the RA/RI/RC, the community will work to determine the resolution, and modify the RA/RI/RC accordingly.
```

Comment:
Would it mean that RI1 should be verified 3 times in a row too?
R1 as all OPNFV installers is tested once right after deployment and then reinstalled. 3 times may secure better results but most of the operations are covered several times (see rally_full).

Reply:
Verification three times was designated to help reduce the possibility of false positives.  The community should have a conversation to determine the most appropriate number of iterations.

Reply:
Yes I see your points. The testing part is good and we may not find false positive. Sometimes a few tempest tests are "falsy" failing but it's mostly due to the SUT selected and the hyperthreading in Functest gates (not in place in RC).